src/node/snapshot_serdes.h

```diff
@@ -161,6 +161,7 @@ namespace ccf
     cd.set(std::move(claims_digest));
     ccf::TxReceiptImpl tx_receipt(
       sig,
+      std::nullopt, // cose
```

Comment:
Shall we have in a snapshot serdes at the moment?

Reply:
Yes, let's be as uniform as possible. Ideally this is step 1 of a 3 step plan:

1. Add COSE signatures as something the application code can use
2. Rebase all internal and tool usage to COSE signatures
3. Offer a configuration option to start new ledgers "COSE only"

Reply:
But this means we have to add a cose signature to snapshot, doesn't it? Do we do it in this PR then?

Reply:
Talked about that, discussed to leave as is for now, will do in subsequent PRs.